include/multipass/constants.h

```diff
@@ -51,14 +51,15 @@ constexpr auto bridged_network_name = "bridged";
 constexpr auto settings_extension = ".conf";
 constexpr auto daemon_settings_root = "local";
 
-constexpr auto petenv_key = "client.primary-name";     // This will eventually be moved to some dynamic settings schema
-constexpr auto driver_key = "local.driver";            // idem
-constexpr auto passphrase_key = "local.passphrase";    // idem
-constexpr auto bridged_interface_key = "local.bridged-network"; // idem
-constexpr auto mounts_key = "local.privileged-mounts"; // idem
-constexpr auto autostart_key = "client.gui.autostart"; // idem
+constexpr auto petenv_key = "client.primary-name";  // This will eventually be moved to some dynamic settings schema
+constexpr auto driver_key = "local.driver";         // idem
+constexpr auto passphrase_key = "local.passphrase"; // idem
+constexpr auto bridged_interface_key = "local.bridged-network";       // idem
+constexpr auto mounts_key = "local.privileged-mounts";                // idem
+constexpr auto autostart_key = "client.gui.autostart";                // idem
 constexpr auto winterm_key = "client.apps.windows-terminal.profiles"; // idem
 constexpr auto hotkey_key = "client.gui.hotkey";                      // idem
+constexpr auto mirror_key = "local.image.mirror";
```

Comment:
Hey @townsend2010! I'm also on the fence about this. On one hand, `image` would make it easier to understand what is that mirror for. But, on the other hand, `simplestreams` would be more descriptive of what Multipass expects as an image. I vote thus for the latter choice.

Making whatever choice, a comment in the code wouldn't hurt at all :)

Reply:
If we don't have strong preference here, I would recommend to stick to current `local.image.mirror`. Since it's not just a key of a configuration, but also an API exposed to the end users. Easy for user to understanding should be a good quality of a product. So I just added a comment here.

However, if we still insist to use the name `simplestreams`, I'm happy to refactor it.

Reply:
Ok, since this is user facing and "image" makes much more sense to an end user than "simplestreams", let's leave it as is.